README.md

```diff
@@ -115,6 +119,50 @@ Commit and push these changes to your local clone on GitHub, and then [create a
 
 We will be notified automatically when you have created your pull request, and providing that everything adheres to nf-core guidelines we will endeavour to approve your pull request as soon as possible.
 
+## Adding a new pipeline specific config
+
+Follow the previous guidelines to create a nf-core/configs, update the pipeline specific documentation, and create the necessary files.
+Replace `<PIPELINE>` with the pipeline name for all the instructions in this guidelines.
+In a similar manner, replace `<PROFILE>` by the name of the profile.
+
+### Pipeline specific documentation
+
+Currently documentation is available for the following pipelines within the specific profile:
+
+* sarek
+  * [MUNIN](docs/pipeline/sarek/munin.md)
+
+### Enabling the specific configs within a specific pipeline
+
+> This has to be done on a fork of the `nf-core/<PIPELINE>` repository.
```

Comment:
```suggestion
:warning:  **This has to be done on a fork of the `nf-core/<PIPELINE>` repository.**
```

Reply:
Just added code suggestion for the next section to have the same format